contracts/Strategy.sol

```diff
@@ -53,54 +127,213 @@ contract Strategy is BaseStrategy {
             uint256 _debtPayment
         )
     {
-        // TODO: Do stuff here to free up any returns back into `want`
-        // NOTE: Return `_profit` which is value generated by all positions, priced in `want`
-        // NOTE: Should try to free up at least `_debtOutstanding` of underlying position
+        liquidityMining.claim();
+        _sell();
+
+        uint256 assets = estimatedTotalAssets();
+        uint256 wantBal = want.balanceOf(address(this));
+
+        uint256 debt = vault.strategies(address(this)).totalDebt;
```

Comment:
we need to be careful with this when downgrading to 0.3.0

Reply:
what changes?